voting/contracts/Voting.sol

```diff
@@ -722,29 +767,25 @@ contract Voting is ReentrancyGuard {
 
 	// Set max percentage of the farm
 	function setMaxPercentageFarm(uint256 _percentage) external {
```

Comment:
@CRYTODEV please also add onlyOwner to this func

Reply:
@michaelnguyen102, good point thanks

Reply:
Resolved